sdks/Cordova.md

```diff
@@ -0,0 +1,138 @@
+# Apache Cordova AeroGear Services
+
+## Motivation
+
+Provide set of the SDK's that can be used for Cordova based development.
+Provide best performance and user experience on the platform.
+
+## Proposed solution 1
+
+Pure javascript solution for the mobile configuration and orchestration 
+that will allow cooperability between JavaScript and Native.
+Mixing both existing Native plugins and pure JavaScript based solutions was identified as the most reasonable aproach.
+Configuration parser and all AeroGear Services specific business code doesn't require any native implementation and can be reused in web/cordova/react native implementations. Individual SDK will may by default be implemented in pure JavaScript or relay on the cordova plugins. 
+
+## Advantages of the solution 1
+
+- Significantly reduced effort to develop and maintain Cordova Support by reusing available components.
+- Aligned with the way cordova works (single plugin to do specific *native* job, integrated with others in JavaScript)
+- Ability to reuse existing cordova plugins that perform native tasks (push notifications, sync, auth)
+- Ability to wrap existing JavaScript solutions and use them without 
+- Ability to select and use only elements that are needed in the app.
+
+See investigation details bellow for more details
+
+## Proposed solution 2
+
+Pure JavaScript implementations (JavaScript SDK) that will abstract from browser support.
```

Comment:
It has both "implementations" and "abstract" and I am not sure what this mean

Reply:
I means that javascript code for example will not assume that `document` variable is available and provide wrapper for ReactNative/Cordova etc. Really sorry for bad wording. Going to explain that carefully.

Reply:
I think the idea is that we write the Cordova SDK in Javascript and use abstractions for APIs that are browser specific (e.g. `fetch` or `axios` for http requests).

Reply:
@pb82 Thank you for that!